i18n/zh/docusaurus-plugin-content-docs/current/installation/standalone.mdx

```diff
@@ -325,7 +325,7 @@ VelaUX 是 KubeVela 的仪表板。 它是在你的集群中运行的 Web 应用
 如果你不使用 KubeVela 的 UI 控制台，可以跳过。
 
 ```shell script
-vela addon enable ~/.vela/addons/velaux
+vela addon enable ~/.vela/addons/velaux serviceType=NodePort
```

Comment:
why this is needed?

Reply:
Align with English version. This is missed at first time. 
PS: In this part, user already have a server with public IP.